homeassistant/components/toon/binary_sensor.py

```diff
@@ -0,0 +1,93 @@
+"""Support for Toon binary sensors."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.toon import ToonEntity
+from homeassistant.components.toon.const import DATA_TOON_CLIENT, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType
+
+DEPENDENCIES = ['toon']
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
+
+
+async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry,
+                            async_add_entities) -> None:
+    """Set up a Toon binary sensor based on a config entry."""
+    toon = hass.data[DATA_TOON_CLIENT][entry.entry_id]
+
+    sensors = []
+
+    sensors.extend([
```

Comment:
No need to call `extend`, you can define it right away in the list declaration 2 lines up

```python
sensors = [
  ToonBinarySensor(…),
]
```

Reply:
Yep, thanks 👍 , addressed.